src/components/Todo/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import EditionForm from './EditionForm';
+import DueDate from './DueDate';
+import Dropdown, { DropdownItem } from '../library/Dropdown';
+
+export default class Todo extends React.Component {
+
+  render() {
+    const { todo } = this.props;
```

Comment:
Here too: 
`const { todo, editing, onUpdate, onCancelEdit, onEdit, onDelete } = this.props;`